src/stories/Tokens/Typography/FontFamily.mdx

```diff
@@ -29,6 +31,64 @@ html {
 }
 ```
 
+## Variable Fonts
+
+Optics supports variable fonts. Variable fonts are a single font file that contains multiple variations of a font. This allows you to adjust various axes that the font makes available such as the weight, width, slant, and other properties.
+
+Optics uses Noto Sans as a default which has three axes of variations: weight, width, and italic. You can adjust these axes using the font-weight, font-stretch, and font-style tokens.
+
+```css
+.my-selector {
+  font-family: var(--op-font-family); // Defaults to "Noto Sans"
+  font-optical-sizing: {auto|none};
+  font-weight: {100 to 900}; // Any value will work, though it is recommended to use the existing weight tokens
```

Comment:
@Jeremy-Walton I recommend showing at least one example of our font-weight tokens here.

Reply:
Updated